GVRf/Sample/gvrf_controls_sample/res/values/strings.xml

```diff
@@ -6,7 +6,7 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software
+ * Unless required by applicable law or agreed to in writing, softwares
```

Comment:
fwiw, "software" was correct.  Software is both singular and plural.  English is weird :)